jobs/claim_unused_prls.go

```diff
@@ -0,0 +1,147 @@
+package jobs
+
+import (
+	"github.com/getsentry/raven-go"
+	"github.com/oysterprotocol/brokernode/models"
+	"github.com/oysterprotocol/brokernode/services"
+	"log"
+	"time"
+)
+
+var ethWrapper = services.EthWrapper
+
+func init() {
+}
+
+func ClaimUnusedPRLs(ethService services.Eth, thresholdTime time.Time) {
+
+	SetEthWrapper(ethService)
+
+	ResendTimedOutGasTransfers(thresholdTime)
+	ResendTimedOutPRLTransfers(thresholdTime)
+
+	ResendErroredGasTransfers()
+	ResendErroredPRLTransfers()
+
+	SendGasForNewClaims()
+	StartNewClaims()
+
+	PurgeCompletedClaims()
+}
+
+func SetEthWrapper(ethService services.Eth) {
+	ethWrapper = ethService
+}
```

Comment:
Using this to tell this file to use the EthMock instead of the real thing, for the tests.